modules/harvest/src/Entity/HarvestRun.php

```diff
@@ -46,15 +46,12 @@
  *   admin_permission = "administer harvest_run",
  *   entity_keys = {
  *     "id" = "id",
- *     "label" = "id",
+ *     "label" = "ID",
```

Comment:
'label' here tells the entity system which field to use *as* a label, not what the label is. We don't really have one, so we should just leave it as id (lower case).

Reply:
Good catch.  Do we even need to specify a label?   Lots of entities don't.

Reply:
Resolved.  I changed it back to id.